### ISSUE TITLE
Commands in command palette not loading initially

### ISSUE THREAD

**Issue by halfst** [srg-kostyrko/obsidian-journal#66]
I'm having an issue where when I start Obsidian the commands for Journals do not show up in the command palette. Because of this, I'm unable to use hotkeys to navigate my daily notes like I'd like to. The only workaround I have found so far is to turn the plugin off and on again, but it's annoying to have to do this every time I start Obsidian.

**Obsidian start-up time breakdown**

Obsidian version: v1.7.1
Installer version: v1.5.3
Operating system: Windows 10 Pro 10.0.19045

- Total startup time: 2,074ms
- Initialization: 240ms
- Vault (1,119 files): 162ms
- Workspace: 209ms
- Core plugins: 18ms
- Community plugins (33 active): 1,445ms
  - Excalidraw
  - Git
  - Copilot
  - Linter
  - Omnisearch
  - QuickAdd
  - Dataview
  - Text Extractor
  - Fleeting Notes Sync
  - Tracker
  - ExcaliBrain
  - Templater
  - Readwise Official
  - Style Settings
  - Quick Switcher++
  - List Callouts
  - Journals
  - Slash Commander
  - Minimal Theme Settings
  - Homepage
  - Tag Wrangler
  - Advanced URI
  - Outliner
  - Settings Search
  - Buttons
  - Workspaces Plus
  - Book Search
  - Calendar
  - Multi Properties
  - Obsidian Columns
  - Hider
  - Footnote Shortcut
  - Auto Link Title

**Comment by srg-kostyrko** [comment]
Could you please check if there are any errors reported in developers console
You can open it from menu View -> Toggle developer tools

**Comment by halfst** [comment]
app.js:1 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'children')
    at t.getSideLeaf (app.js:1:2292574)
    at t.getLeftLeaf (app.js:1:2291275)
    at JournalManager.placeCalendarView (plugin:journals:5939:38)
    at JournalPlugin.eval (plugin:journals:6769:30)
    at Generator.next (<anonymous>)
    at fulfilled (plugin:journals:25:58)

**Comment by srg-kostyrko** [comment]
I have tired reproducing problem on 1.7 version and it still work for me without issues

do you have something in your plugins that heavily changes layout (like removing left sidebar)?

**Comment by halfst** [comment]
Nothing that removes the sidebar. When I first open Obsidian the Journals calendar isn't available in the sidebar, but after I turn off and on the plugin it shows up in the sidebar. I tried moving the calendar to the right sidebar and I get the same issue. I also tried uninstalling and reinstalling the plugin, but that made no difference.

I think I figured out the issue though. I have the Homepage plugin set to load a specific workspace every time I open Obisidian, and the calendar wasn't on that workspace. I added the calendar to the workspace and saved it and now it loads fine.
